gsplat/cuda/csrc/relocation.cu

```diff
@@ -42,14 +42,16 @@ std::tuple<torch::Tensor, torch::Tensor> compute_relocation_tensor(
     torch::Tensor &binoms,
     const int n_max
 ) {
+    DEVICE_GUARD(opacities);
+    CHECK_INPUT(opacities);
+    CHECK_INPUT(scales);
+    CHECK_INPUT(ratios);
+    CHECK_INPUT(binoms);
+    torch::Tensor new_opacities = torch::zeros_like(opacities);
+    torch::Tensor new_scales = torch::zeros_like(scales);
```

Comment:
These could be `torch::empty_like` as the following kernels would write values to it anyway. Using empty_like could save up one kernel launch for filling in zeros.